oop/src/views/MoveInfoPage.vue

```diff
@@ -362,13 +379,33 @@ export default {
     getMinAddressLines() {
       return MIN_ADDRESS_LINES;
     },
+    setFieldsToNull() {
+      // Remove all current address lines
+      for (let i=0; i<this.addressLines.length; i++) {
+        setTimeout(() => {
+          this.removeAddressField();
+        }, 0);
+      }
```

Comment:
Why is there a timeout for each one of the address lines? Can this be wrapped in one timeout? eg:
```javascript
setTimeout(() => {
  for (let i=0; i<this.addressLines.length; i++) {
    this.removeAddressField();
  }
}, 0);
```